### ISSUE TITLE
Can't connect to dashboard after login. SSL and CORS errors, API working

### ISSUE THREAD

**Issue by omkar-tenkale** [parse-community/parse-dashboard#1296]
Make sure these boxes are checked before submitting your issue -- thanks for reporting issues back to Parse Dashboard!

- [x] You're running version >=2.0.3 of Parse Dashboard.

- [x] You're running version >=2.7.1 of Parse Server.

- [x] You've searched through [existing issues](https://github.com/ParsePlatform/parse-dashboard/issues?utf8=%E2%9C%93&q=). Chances are that your issue has been reported or resolved before.

#### Environment Setup
Windows 10
2.0.3 of Parse Dashboard.
2.7.1 of Parse Server.
#### Steps to reproduce
```
C:\parse-server>node -v
v10.16.3


C:\parse-server>npm start

> parse-server-example@1.4.0 start C:\parse-server
> node index.js

DATABASE_URI not specified, falling back to localhost.
parse-server-example running on port 1337.
info: Parse LiveQuery Server starts running

```

**now starting parse dashboard on new window**

```

C:\parse-server>parse-dashboard  --appId myAppId --masterKey world --serverURL "https://localhost:1337/parse" --appName optionalName --allowInsecureHTTP true
The dashboard is now available at http://0.0.0.0:4040/
```

now opening in browser:
http://localhost:1337/parse/serverInfo
response:  
{"error":"unauthorized"}

also testing from 
http://localhost:1337/test
works well.

but after visiting http://localhost:4040
it forwards to 
http://localhost:4040/apps/optionalName/browser
and a blank page appears.

![Screenshot (76)](https://user-images.githubusercontent.com/39808155/65125504-169afa80-da13-11e9-9970-bd3c637473e5.png)
![Screenshot (77)](https://user-images.githubusercontent.com/39808155/65125516-1dc20880-da13-11e9-80b8-acaa396c3fba.png)
![Screenshot (78)](https://user-images.githubusercontent.com/39808155/65125555-33373280-da13-11e9-8cb3-0081352f8119.png)
![Screenshot (79)](https://user-images.githubusercontent.com/39808155/65125567-37635000-da13-11e9-99dc-9a2e6365f7e8.png)

#### Logs/Trace
Note: If you get a browser JS error please run `npm run dev`. This will provide source maps and a much more useful stack trace.

**Comment by omkar-tenkale** [comment]
took reference from this site
https://tecadmin.net/setup-parse-dashboard-on-ubuntu/

the comment there also has the similar problem

**Comment by omkar-tenkale** [comment]
**my index.js**

```
// Example express application adding the parse-server module to expose Parse
// compatible API routes.

var express = require('express');
var ParseServer = require('parse-server').ParseServer;
var path = require('path');

var databaseUri = process.env.DATABASE_URI || process.env.MONGODB_URI;

if (!databaseUri) {
  console.log('DATABASE_URI not specified, falling back to localhost.');
}

var api = new ParseServer({
  databaseURI: databaseUri || 'mongodb://localhost:27017/dev',
  cloud: process.env.CLOUD_CODE_MAIN || __dirname + '/cloud/main.js',
  appId: process.env.APP_ID || 'myAppId',
  masterKey: process.env.MASTER_KEY || 'world', //Add your master key here. Keep it secret!
  serverURL: process.env.SERVER_URL || 'http://localhost:1337/parse',  // Don't forget to change to https if needed
  liveQuery: {
    classNames: ["Posts", "Comments"] // List of classes to support for query subscriptions
  }
});
// Client-keys like the javascript key or the .NET key are not necessary with parse-server
// If you wish you require them, you can set them as options in the initialization above:
// javascriptKey, restAPIKey, dotNetKey, clientKey

var app = express();

// Serve static assets from the /public folder
app.use('/public', express.static(path.join(__dirname, '/public')));

// Serve the Parse API on the /parse URL prefix
var mountPath = process.env.PARSE_MOUNT || '/parse';
app.use(mountPath, api);

// Parse Server plays nicely with the rest of your web routes
app.get('/', function(req, res) {
  res.status(200).send('I dream of being a website.  Please star the parse-server repo on GitHub!');
});

// There will be a test page available on the /test path of your server url
// Remove this before launching your app
app.get('/test', function(req, res) {
  res.sendFile(path.join(__dirname, '/public/test.html'));
});

var port = process.env.PORT || 1337;
var httpServer = require('http').createServer(app);
httpServer.listen(port, function() {
    console.log('parse-server-example running on port ' + port + '.');
});

// This will enable the Live Query real-time server
ParseServer.createLiveQueryServer(httpServer);

```

**Comment by omkar-tenkale** [comment]
**and config.json**

`{
  "apps": [
    {
      "serverURL": "http://localhost:1337/parse",
      "appId": "myAppId",
      "masterKey": "world",
      "appName": "myAppId"
    }
  ],
  "users": [
      {
           "user":"username",
           "pass":"password"
      }
  ]
}`

**Comment by omkar-tenkale** [comment]
config.json
```
{
  "apps": [
    {
      "serverURL": "http://localhost:1337/parse",
      "appId": "myAppId",
      "masterKey": "world",
      "appName": "myAppId"
    }
  ],
  "users": [
      {
           "user":"username",
           "pass":"password"
      }
  ]
}
```

**Comment by omkar-tenkale** [comment]
**Solved**

I don't know what the problem was but i solved it this way
1. Downloaded this repo as zip (previously installed via npm in parse-server folder)
2. Extracted the zip to C:/ 
3. edited the file C:\parse-dashboard\Parse-Dashboard\parse-dashboard-config.json
```
{
  "apps": [{
    "serverURL": "http://localhost:1337/parse",
    "appId": "myAppId",
    "masterKey": "world",
    "appName": "myapp"
      }]
}
```
4.opened new cmd window with admin privilage
5.cd to C:\parse-dashboard\Parse-Dashboard\ with command 
```
C:\Windows\system32>cd /d C:/parse-dashboard
C:\parse-dashboard>cd Parse-Dashboard
```
6.ran command `parse-dashboard --config parse-dashboard-config.json`
7.result 
```
C:\parse-dashboard\Parse-Dashboard>parse-dashboard --config parse-dashboard-config.json
The dashboard is now available at http://0.0.0.0:4040/
```
8.open url `http://localhost:4040/`
works in chrome and opera but still a blank page in mozilla firefox probably because addons/about:config tweaks i did
